src/kernels/errors/kernelErrorHandler.ts

```diff
@@ -222,23 +224,35 @@ export abstract class DataScienceErrorHandler implements IDataScienceErrorHandle
 
         // Jupyter kernels, non zmq actually do the dependency install themselves
         if (isCancellationError(err)) {
+            traceVerbose(`Cancelled by user`);
```

Comment:
This wouldn't be very clear in a log file, can you add more context - even just a tag like [kernelErrorHandler]

Reply:
Remvoed that, i think its unnecessary.